servers/display_server.h

```diff
@@ -504,6 +512,8 @@ class DisplayServer : public Object {
 	virtual bool window_get_flag(WindowFlags p_flag, WindowID p_window = MAIN_WINDOW_ID) const = 0;
 
 	virtual void window_request_attention(WindowID p_window = MAIN_WINDOW_ID) = 0;
+	virtual void window_set_taskbar_progress_value(float p_value, WindowID p_window = MAIN_WINDOW_ID) = 0;
+	virtual void window_set_taskbar_progress_state(ProgressState p_state, WindowID p_window = MAIN_WINDOW_ID) = 0;
```

Comment:
```suggestion
	virtual void window_set_taskbar_progress_value(float p_value, WindowID p_window = MAIN_WINDOW_ID) {};
	virtual void window_set_taskbar_progress_state(ProgressState p_state, WindowID p_window = MAIN_WINDOW_ID) {};
```

It won't be supported on most platforms, so there's no need to make it pure virtual and adding dummy implementations in all display servers, dummy base and Windows implementation should be more clear.